resources/assets/js/components/FriendsList.vue

```diff
@@ -129,13 +129,16 @@
                     text: 'This will delete @' + friend.username + ' from your list of friends',
                     type: 'warning',
                     showCancelButton: true
-                }, function () {
+                }, () => {
                     vm.$http.delete('/api/conferences/' + vm.conferenceId + '/' + vm.key + '/' + friend.id)
-                        .then(function () {
-                            vm.list.$remove(friend)
-                        });
+                        .then(() => { vm.list.$remove(friend) });
                 });
             },
+
+            metFriend: function (friend) {
+                this.$http.patch('/api/conferences/' + this.conferenceId + '/' + this.key + '/' + friend.id, { met: ! friend.met })
```

Comment:
It currently uses the existing endpoint and toggles the `met` flag from `true` to `false`, so the call simply sets the inverse of what it is currently.